### PR TITLE
prefix redux

### PR DESCRIPTION
**Pull request by mcovarr**

### CODE REVIEW

File: SPEC.md [hunk 1]
```diff
@@ -2550,16 +2550,19 @@ Integer xlen = length(xs) # 3
 Integer ylen = length(ys) # 3
 Integer zlen = length(zs) # 0
```
Comment (katevoss) [inline SPEC.md]: Missing ``` here

Comment (mcovarr) [inline SPEC.md]: whoops good catch

**Comment by kshakir** [comment]
👍

**Comment by ruchim** [comment]
:+1:

**Comment by geoffjentry** [comment]
👍

**Pull request merged**